ocp4/profiles/cis.profile

```diff
@@ -60,6 +60,7 @@ selections:
     - api_server_profiling
   # 1.2.22 Ensure that the --audit-log-path argument is set
     - api_server_audit_log_path
+    - kube_api_server_audit_log_path
```

Comment:
I think we should instead explicitly create an extra rule that checks  for the openshift apiserver.

Something such as `openshift_apiserver_audit_log_path`. The rest of the rules that start with `api_server` check the kube-apiserver already.

Reply:
Yeah, I'm going to redo some stuff with these in mind. Thanks